src/logics/compound-v3/logic.borrow.test.ts

```diff
@@ -42,7 +42,7 @@ describe('CompoundV3 BorrowLogic', function () {
         const { marketId, output } = fields;
         const market = getMarket(chainId, marketId);
 
-        expect(routerLogic.to).to.eq(market.cometAddress);
```

Comment:
please also help fix the `cometAddress` in test/logics/compound-v3/helpers.ts

Reply:
fixed, sorry for not notice `test` part